integration-test/src/python/local_test_runner/main.py

```diff
@@ -190,34 +195,35 @@ def submitTopology(heronCliPath, testCluster, testJarPath, topologyClassPath,
       '%s' % (outputFile),
       '%d' % (len(TEST_INPUT))
   ]
-  logging.info("Submitting topology: ")
+  logging.info("Submitting topology: %s", splitcmd)
   logging.info(splitcmd)
```

Comment:
Where do you see the other occurrence? I'm not seeing it. Below we log topologyName.